MatchGame/Deck.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MatchGame
+{
+    public class Deck : IDeck
+    {
+        private List<Card> _cards;
+
+        public Deck()
+        {
+            _cards = BuildStandardPack();
```

Comment:
It looks like maybe there was a case for using Composition here, through a Pack class which would have generated the pack(s) in the deck on construction of the object and made the Deck more SRP for use in the game? The Game doesn't need to know how the deck is built!?